code/modules/materials/material_recipes.dm

```diff
@@ -40,7 +40,7 @@
 		new/datum/stack_recipe("purple comfy chair", /obj/structure/bed/chair/comfy/purp, 2, one_per_turf = 1, on_floor = 1), \
 		new/datum/stack_recipe("green comfy chair", /obj/structure/bed/chair/comfy/green, 2, one_per_turf = 1, on_floor = 1), \
 		))
-
+	recipes += new/datum/stack_recipe("brass knuckles", /obj/item/weapon/brassknuckles, 1, time = 10, one_per_turf = 0, on_floor = 1)

```

Comment:
all it does is force you to be on a floor when building it, it's literally pointless for bats and other items you can pick up